public_profiler/v3.5-DD/ogl/skills.js

```diff
@@ -154,7 +154,6 @@ for (var skill in classSkills)
 }
 
 var baseHelpURL = "http://www.d20srd.org/srd/";
-var baseWikiURL = "https://sidequestors.skydragoness.com/mediawiki/view/";
```

Comment:
I would be happier if baseWikiURL never entered the history, i.e. if this commit were squashed with the one that introduced baseWikiURL, thereby removing it.  Our particular installation should not be part of the git history ...